src/js/components/ToggleCheckboxList/ToggleCheckboxList.tsx

```diff
@@ -79,23 +100,24 @@ const ToggleCheckboxList: React.FunctionComponent<Props> = ({
     setHeaderStatus(currHeaderStatus);
   }
 
-  const toggleSubheadingRows = (heading: string, idx: number) => {
-    let rowData= [];
+  const toggleSubheadingRows = (heading: string, subheading: string, idx: number) => {
+    let rowData = [];
+    const headerName = `${heading}-${subheading}`;
 
-    if (heading in headerStatus && headerStatus[heading]['activeCount'] > 0) {
+    if (headerName in headerStatus && headerStatus[headerName]['activeCount'] >= 0) {
```

Comment:
Optional, but I think you can shorten this to 
```suggestion
    if (headerStatus[headerName]?.['activeCount'] >= 0) {
```